index.html

```diff
@@ -0,0 +1,101 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kamulete training center</title>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">    
+    <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav class="twitter">
+            <a href="#"><i class="fa-brands fa-square-twitter" width="10px"></i></a>
+            <a href="#">Whatsap</a>
+            <a href="#">training</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="navlogo">
+            <a class="img-logo" href="index.html"><img src="img-folder/logo-image---00.png" alt=""></a>
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="#programs">program</a></li>
+                <li><a class="menu-item" href="#ourTeacher">Teachers</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+            </ul>    
+        </nav>
+    </header>
+        <section class="kamuclass">
+            <p class="hello">" Hello! Are you intersted to Speak many languages?"</p>
+            <h1>DO YOU WANT TO KNOW<br> MANY LANGUAGES IN <br>2023?</h1>
+            <p class="allP">we teach how to speak fluently each language <br> like french ,english,spanish,arbic, and more.<p>
+            <p class="stating-date">2023.10.25(MON)~10(FRI)</p>
+            <p class="allP">@ National of RWAN we get stated in 2023 <p>
+        </section>
+        <section id="programs" class="programsclass">
+            <h2>programs</h2>
+            <hr class="linebar">
+    <div class="bloc-programs">
+        <div class="bloc1">
+            <i class="fa-solid fa-person-chalkboard fa-3x"></i>
+            <h3>4 months class</h3>
+            <p>Is a program that you can learn with other student everyday.</p>
+        </div>
+        <div  class="bloc1">
+            <i class="fa-solid fa-chalkboard-user fa-3x"></i>
+            <h3>12 months class</h3>
+            <p>This program is 12 months because hours are limited.</p>
+        </div>
+        <div  class="bloc1">
+            <i class="fa-solid fa-children fa-3x"></i>
+            <h3>Night <br>schedule</h3>
+            <p>in the night program a student must learn during night time.</p>
+        </div>
+        <div  class="bloc1">
+            <i class="fa-solid fa-home fa-3x"></i>
+            <h3>weekly <br>home</h3>
+            <p>In this program the studies is scheduled per week.</p>
+        </div>
```

Comment:
[OPTIONAL] Please check again the design in the template and increase the size of this sections title and reduce the size of the link in its bottom 

![capstone programs title size](https://user-images.githubusercontent.com/59707859/225984780-b1b5aa30-9cf5-457b-92c9-c36435399193.PNG)